modules/gapi/src/api/kernels_video.cpp

```diff
@@ -57,5 +57,10 @@ GMat BackgroundSubtractor(const GMat& src, const BackgroundSubtractorParams& bsp
     return GBackgroundSubtractor::on(src, bsp);
 }
 
+GMat KalmanFilter(const GMat& m, const cv::GOpaque<bool>& have_m, const GMat& c, const KalmanParams& kp)
```

Comment:
So here you have `KalmanParams` passed as operation parameters, does it needs to be declared as a `CompileArg` above?

Reply:
Could you please clarify where KalmanParams was declared as a `CompileArg`?

Reply:
Actually, a significantly valid remark!
[modules/gapi/include/opencv2/gapi/video.hpp:388](https://github.com/opencv/opencv/pull/18869/files#diff-2287fa1644891ed2c702b012209de16f354dd23291b78a97a794a5949d060fc4R388)
We don't use the initialization via CompileArgs anymore, so this overload for CompileArg should be removed.
The same should be applied for the `BackgroundSubtractorParams` as well!

Reply:
Done for Kalman.

Reply:
Would you apply this for BGSub in a separate PR? Or I can do this, if you'd like.

Reply:
I'll do this later.